pynitrokey/libnk.py

```diff
@@ -508,6 +508,33 @@ def enable_firmware_update(self, password):
 
         return self.api.NK_enable_firmware_update(c_enc(password))
 
+    @ret_code
+    def unlock_encrypted_volume(self, password):
+        """Unlock the storage device's encrypted volume"""
+        return self.api.NK_unlock_encrypted_volume(c_enc(password))
+
+    @ret_code
+    def lock_encrypted_volume(self):
+        """Lock the storage device's encrypted volume"""
+        return self.api.NK_lock_encrypted_volume()
+
```

Comment:
I think here the full device's name should be used:
`storage` -> `NK Storage`

Let's correct that in the next PR.